roles/edpm_nova/molecule/default/verify.yml

```diff
@@ -40,3 +40,17 @@
         - "Copying /var/lib/kolla/config_files/nova-blank.conf to /etc/nova/nova.conf"
         - "Copying /var/lib/kolla/config_files/01-nova.conf to /etc/nova/nova.conf.d/01-nova.conf"
         - "Copying /var/lib/kolla/config_files/02-nova-log.conf to /etc/nova/nova.conf.d/02-nova-log.conf"
+
+    - name: Lookup the nova user
+      ansible.builtin.shell:
```

Comment:
There is a comment from @ielod this in https://github.com/openstack-k8s-operators/edpm-ansible/pull/446#discussion_r1358369686 that I will fix quickly